packages/react-native-select-pro/src/components/select/index.tsx

```diff
@@ -20,6 +20,11 @@ import type {
 import { OptionsList } from '../options-list';
 import { SelectControl } from '../select-control';
 
+const logDataError = () => {
+    // eslint-disable-next-line no-console
+    console.error('You must pass array in the options prop');
```

Comment:
Lets create global errors object or enum eg.:
ERRORS = {
NO_ARRAY_OPTIONS: 'You must pass array in the options prop'
...other errors in the future
}